### DISCUSSION TITLE
GitHub Dark Colorblind Dimmed

### DISCUSSION THREAD

**Discussion by vanyauhalin** [primer/github-vscode-theme#341] [Ideas]
Many thanks to the GitHub team for the colorblind themes. Even though I'm not colorblind, I still prefer to use colorblind themes, as the colors are more muted, which allows me to better focus on the code. But, unfortunately, the dark theme has a very dark background, which makes the text look brighter and more defiant. Therefore, I decided to borrow black, white, gray colors from GitHub Dark Dimmed. I'm not sure if my solution is correct in terms of colorblindness, but for me, it was a great solution. You may be able to add this preset as a new theme named GitHub Dark Colorblind Dimmed.

GitHub Dark Colorblind Dimmed:

<img width="1136" alt="GitHub Dark Colorblind Dimmed" src="https://user-images.githubusercontent.com/56481109/204260609-6f1d85fb-3e20-4a64-8de9-78ac9a4b5556.png">

GitHub Dark Colorblind:

<img width="1136" alt="GitHub Dark Colorblind" src="https://user-images.githubusercontent.com/56481109/204263686-baea6166-efeb-4448-a4bd-ac482488fb37.png">

**Comment by simurai** [comment]
Thanks for the feedback. At the moment there is no plan for an adding `GitHub Dark Colorblind Dimmed` theme as we try to mirror [github.com's appearance setting](https://github.com/settings/appearance).

> Therefore, I decided to borrow black, white, gray colors from GitHub Dark Dimmed. I'm not sure if my solution is correct in terms of colorblindness, but for me, it was a great solution. 

Yes, customizing within your VS Code settings is a great way to make these tweaks. All monochrome colors (white, gray, black) are not changed for the colorblind themes, so using the `Dimmed` colors should work fine. 👍

**Comment by simurai** [comment]
FYI.. I converted this into a discussion.

**Comment by vanyauhalin** [comment]
This is no longer relevant since I have decided to shift my focus toward developing [my theme](https://github.com/vanyauhalin/moondusttheme/). Anyway, it seems to me a bad idea to implement the theme above. It appears to conflict with the purpose of the colorblind theme itself.
